hpy/devel/__init__.py

```diff
@@ -130,6 +130,11 @@ def __bootstrap__():
     env_debug = environ.get('HPY_DEBUG_CONTEXT')
     is_debug = env_debug is not None and (env_debug == "1" or __name__ in env_debug.split(","))
     ext_filepath = resource_filename(__name__, {ext_file!r})
+    if 'HPY_LOG' in environ:
+        if is_debug:
+            print("Loading {module_name!r} in HPy universal mode with a debug context")
+        else:
+            print("Loading {module_name!r} in HPy universal mode")
```

Comment:
I'm fine using the `print` for now to avoid blocking this PR forever, but eventually we should probably use `logging` module for that, and `HPY_LOG` should be used to select the desired log level